tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore5IastTests.cs

```diff
@@ -567,6 +567,40 @@ await VerifyHelper.VerifySpans(spansFiltered, settings)
                           .DisableRequireUniquePrefix();
     }
 
+    [Trait("Category", "EndToEnd")]
+    [Trait("RunOnWindows", "True")]
+    [SkippableTheory]
+    [InlineData("Authorization", "Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==")]
+    [InlineData("Authorization", "basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==")]
+    [InlineData("Authorization", "    bAsic    QWxhZGRpbjpvcGVuIHNlc2FtZQ==")]
+    [InlineData("Authorization", "digest QWxhZGRpbjpvcGVuIHNlc2FtZQ==")]
+    public async Task TestIastInsecureAuthProtocolRequest(string header, string data)
+    {
+        var filename = IastEnabled ? "Iast.InsecureAuthProtocol.AspNetCore5.IastEnabled" : "Iast.InsecureAuthProtocol.AspNetCore5.IastDisabled";
+
+        // Test on basic or digest
+        if (IastEnabled)
+        {
+            var scheme = data.Trim(' ').Split(' ')[0];
+            filename += scheme.Equals("basic", StringComparison.OrdinalIgnoreCase) ? ".BasicAuth" : ".DigestAuth";
```

Comment:
Instead of doing this, I'd add the test "name" as the first argument of the function, because it's not clear what's the difference between the cases